### ISSUE TITLE
Serial?

### ISSUE THREAD

**Issue by VladimirAkopyan** [atlury/Intel-Edison-OS-Images#1]
Hello, i am recently exploring custom images for Edison, and stumbled across this! It's reall yexciting to see mainline support for intel! One thing I am not clear about - Does the 4.8 RC4 include support for serial?

**Comment by atlury** [comment]
Yes it supports serial. Meanwhile we will have a stable kernel release soon, just ironing out some SPI issues here...be patient

https://github.com/andy-shev/linux/issues/1

**Comment by VladimirAkopyan** [comment]
Is there any chance you could also put together a 64 bit image? It's quite useful, for example .Net core only works with linux 64 bit (god knows why)

**Comment by atlury** [comment]
Yes I will put together a 64 bit ubuntu 16.0.4 or 16.10 with a mainline kernel. Please be patient
